invenio/values.yaml

```diff
@@ -131,6 +131,13 @@ redis:
   enabled: true
   image: "redis:7"
   host: ""
+  persistence:
+    enabled: true
+    name: "redis-volume"
+    access_mode: ReadWriteMany
```

Comment:
Minor: it is probably enough `ReadWriteOnce`, given that it is not shared. Anyway, we can keep it like this.